pkgs/tools/misc/todo/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, stdenv
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "todo";
+  version = "2.4";
+
+  src = fetchFromGitHub {
+    owner = "sioodmy";
+    repo = pname;
+    rev = "${version}";
+    sha256 = "Z3kaCNZyknNHkZUsHARYh3iWWR+v//JhuYoMIrq54Wo=";
+  };
+
+  cargoSha256 = "82xB+9kiLBwCE6yC3tlmgzJJgA1cMDq6Mjc48GBZ9B8=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl ];
+
+  meta = with lib; {
+    description = "Simple todo cli program written in rust";
+    homepage = "https://github.com/sioodmy/todo";
+    license = licenses.gpl3;
```

Comment:
You should use `gpl3Only` or `gpl3Plus` instead

Reply:
Should I open a new pull request or what?

Reply:
No, you can force push to PR as you've already done :)